fontdrasil/src/orchestration.rs

```diff
@@ -4,15 +4,22 @@ use std::{
     collections::HashSet,
     fmt::{Debug, Display},
     hash::Hash,
-    sync::Arc,
 };
 
 pub const MISSING_DATA: &str = "Missing data, dependency management failed us?";
 
+/// Identifies all items of the same group, typically enum variant, of an identifer
+///
+/// String because it's enormously easier to understand debug output that way
+pub type IdentifierDiscriminant = &'static str;
```

Comment:
This could just as easily be a type with a `Display` impl?

Reply:
It could be but it would make use of the value more awkward for no obvious benefit?

Reply:
I played with this locally and it seemed to make my life worse so I'm inclined not to do it.